prov/verbs/src/verbs_mr.c

```diff
@@ -215,10 +215,8 @@ int fi_ibv_mr_cache_add_region(struct ofi_mr_cache *cache,
 	md->mr_fid.fid.ops = &fi_ibv_mr_cache_fi_ops;
 	md->entry = entry;
 
-	return fi_ibv_mr_reg_common(md, IBV_ACCESS_LOCAL_WRITE |
-			IBV_ACCESS_REMOTE_WRITE | IBV_ACCESS_REMOTE_ATOMIC |
-			IBV_ACCESS_REMOTE_READ, entry->info.iov.iov_base,
-			entry->info.iov.iov_len, NULL);
+	return fi_ibv_mr_reg_common(md, ofi_mr_access_map[entry->info.mr_access],
```

Comment:
This is wrong.  ofi_mr_access_map is returning FI_xxx permissions (FI_READ, FI_SEND, etc.), not libibverbs access.